golem-worker-executor-base/src/storage/mod.rs

```diff
@@ -0,0 +1,17 @@
+// Copyright 2024 Golem Cloud
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+pub mod blob;
+pub mod indexed;
+pub mod keyvalue;
```

Comment:
Might be nice to have a sqllite version of this (and maybe `blob`) just for local development.

Reply:
I agree, just did not want to spend more time on implementing new backends now.